resources/views/dashboard.blade.php

```diff
@@ -178,7 +178,7 @@ class="table table-striped snipe-table"
 </div> <!--/row-->
 <div class="row">
     <div class="col-md-6">
-        <div class="box box-default" style="min-height: 400px;">
```

Comment:
Why are you removing this?

Reply:
As min-height is set when you collapse the box it doesn't collapse only hides the content, if there is a better way i'm happy to change it back

Reply:
Hm. Yeah, I can see what you mean. Unfortunately that pie chart can show up funny on some browsers if a min height isn't set. Maybe put it inside of a div that has a min-height (within the box-default div)

Reply:
How about attaching the min-height to the box-body instead of the entire box?

Reply:
Give it a try - if it works, I'm cool with it

Reply:
i pushed the change aways :D

Reply:
:-P